news/migrations/0014_auto__add_smsmessage.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
```

Comment:
What's the timeline on moving basket to django 1.7+ with the new style of migrations?